### PR TITLE
Matecoin trades

### PR DESCRIPTION
**Pull request by MasterpieceElbow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,45 @@
-# Python boilerplate for GitHub tasks
+# Matecoin trades
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+You have made some trades with new cryptocurrency "Matecoin".
+In order to calculate profit you loaded json-file `trades.json` with 
+trades description. Content of that file is `JsvsScript Object`
```
Comment (danylott) [inline README.md:7]: Better here list of Javascript Objects

File: app/trades.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (danylott) [inline app/trades.json:1]: You'd better create it as list of objects, not with this date formatting using `_`.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,45 @@
...
+which is very similar to 
+python `dict`, where keys are dates of trades and values are
```
Comment (danylott) [inline README.md:9]: And so - here are without `keys`

**Review by danylott — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,4 +1,45 @@
...
+    "bought": null,
+    "sold": "0.00058",
+    "matecoin_price": "77830.83"
+  }
+]
+```
+
+Write `calculate_profit` function, that takes name of file with
+trades information, converts inside data into python native data
+structure, counts money profit and current coin account, dumps
```
Comment (IvanRamyk) [inline README.md:33]:
```suggestion
structure, counts money profit in dollars and current coin account, dumps
```

File: README.md [hunk 5]
```diff
@@ -1,4 +1,45 @@
-# Python boilerplate for GitHub tasks
+# Matecoin trades
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+You have made some trades with new cryptocurrency "Matecoin".
```
Comment (IvanRamyk) [inline README.md:5]:
```suggestion
You have made some trades with the new cryptocurrency "Matecoin".
```

File: README.md [hunk 6]
```diff
@@ -1,4 +1,45 @@
...
+  },
+  {
+    "bought": null,
+    "sold": "0.00058",
+    "matecoin_price": "77830.83"
+  }
+]
+```
+
+Write `calculate_profit` function, that takes name of file with
```
Comment (IvanRamyk) [inline README.md:31]:
```suggestion
Write `calculate_profit` function, which takes the name of the file with
```

**Review by IvanRamyk — APPROVED** [review]

**Pull request merged**